lib/swaps/Swaps.ts

```diff
@@ -433,6 +433,29 @@ class Swaps extends EventEmitter {
 
     const { makerCurrency, makerAmount, takerCurrency, takerAmount } = Swaps.calculateMakerTakerAmounts(quantity, price, isBuy, requestBody.pairId);
 
+    const makerSwapClient = this.swapClientManager.get(makerCurrency)!;
+    if (!makerSwapClient) {
```

Comment:
It's interesting that we need this extra logic here since we already have a passing test in master that covers this:
https://github.com/ExchangeUnion/xud/blob/master/test/jest/integration/Swaps.spec.ts#L123

Reply:
Yes it's only needed due to what I described here: https://github.com/ExchangeUnion/xud/pull/1072#discussion_r299194032

It's basically just to check for a null value.